documentation/asciidoc/computers/camera/rpicam_apps_intro.adoc

```diff
@@ -20,9 +18,11 @@ For this reason Raspberry Pi supplies a small set of example `rpicam-apps`. Thes
 * `rpicam-raw`: Captures raw (unprocessed Bayer) frames directly from the sensor.
 * `rpicam-detect`: Not built by default, but users can build it if they have TensorFlow Lite installed on their Raspberry Pi. Captures JPEG images when certain objects are detected.
 
-Users can create their own rpicam-based applications with custom functionality to suit their own requirements. The https://github.com/raspberrypi/rpicam-apps[`rpicam-apps` source code] is freely available under a BSD-2-Clause licence.
+Recent versions of Raspberry Pi OS include the five basic `rpicam-apps`, so you can record images and videos using a camera even on a fresh Raspberry Pi OS installation.
```

Comment:
"...include the five basic `rpicam-apps` pre-installed, so..." ?

Reply:
doesn't "include" render "pre-installed" redundant? I could also replace "include" with "pre-install" but that seems vaguely less correct.